doc/source/reference/random/index.rst

```diff
@@ -18,41 +85,36 @@ to use those sequences to sample from different statistical distributions:
   distribution (such as uniform, Normal or Binomial) within a specified
   interval.
 
-Since Numpy version 1.17.0 the Generator can be initialized with a
-number of different BitGenerators. It exposes many different probability
-distributions. See `NEP 19 <https://www.numpy.org/neps/
-nep-0019-rng-policy.html>`_ for context on the updated random Numpy number
-routines. The legacy `RandomState` random number routines are still
-available, but limited to a single BitGenerator. See :ref:`new-or-different` 
+In short, BitGenerators are the parts responsible for the randomization
+and the state, whereas Generators are responsible for utilizing these
+randomized inputs.
+
+The Generators can be initialized with a number of different BitGenerators.
+They expose many different probability distributions. For the context
+and the design decisions, we refer to `NEP 19 <https://www.numpy.org/neps/
+nep-0019-rng-policy.html>`_ . The legacy `RandomState` random number routines
+are still available, but limited to a single BitGenerator. See :ref:`new-or-different` 
 for a complete list of improvements and differences from the legacy
 ``RandomState``.
 
 For convenience and backward compatibility, a single `RandomState`
 instance's methods are imported into the numpy.random namespace, see
-:ref:`legacy` for the complete list.
-
-.. _random-quick-start:
-
-Quick Start
------------
-
-Call `default_rng` to get a new instance of a `Generator`, then call its
-methods to obtain samples from different distributions.  By default,
-`Generator` uses bits provided by `PCG64` which has better statistical
-properties than the legacy `MT19937` used in `RandomState`.
+:ref:`legacy` for the complete list. At this point, we have established
+the `Generator` based way-of-working still the legacy API is available.
+While this might seem like, we offer two options, it is quite the
+contrary. We sincerely hope that we minimized the annoyance during
+these changes, however we also hope that users will eventually stop
+using the legacy API which would allow us deprecate and remove the
+legacy API. 
```

Comment:
```suggestion
:ref:`legacy` for the complete list. While the legacy API is still available,
it will eventually be deprecated in the future, so we strongly advise using
the new API instead of the legacy API.
```
The paragraph was hard to understand, so I replaced it.
```

Reply:
what was hard to understand?

Reply:
> what was hard to understand?

At least the first sentence is hard to parse ("we have established [x] still the ..."). I also don't think we should try to "sincerely hope" and frame a fact like its opposite - we _are_ offering two options, but we want (not hope) to transition from one to the other. If Matteo hadn't already posted his suggestion, I would have proposed something very similar.